config/routes.rb

```diff
@@ -314,6 +321,7 @@
             post :make_visible
           end
         end
+        # concerns :issueable
```

Comment:
can this commented line be removed now?

Reply:
Just about to put it back in in the next PR so will leave it there for now.